tests/test_x86.py

```diff
@@ -25860,6 +25860,181 @@ def test_PMINUB_2(self):
         self.assertEqual(cpu.XMM0, 0)
         self.assertEqual(cpu.XMM1, 0)
 
+    def test_PADDD(self):
+        ''' Instruction PADDD
+            Groups: sse2
+            0x8065f84:	paddd    xmm2, xmm7
+        '''
+        mem = Memory32()
+        cpu = I386Cpu(mem)
+        mem.mmap(0x08065000, 0x1000, 'rwx')
+        mem[0x08065f84] = '\x66'
+        mem[0x08065f85] = '\x0f'
+        mem[0x08065f86] = '\xfe'
+        mem[0x08065f87] = '\xd7'
+        cpu.EIP = 0x8065f84
```

Comment:
1. Is there a way to assign multiple bytes at once? E.g. `mem[a:b] = 'aaaa'
2. It would be easier to follow to have address hardcoded once into a variable and then just use indexing/offsets like:
```python
addr = 0x08065f84
mem[addr]
mem[addr+1]
```